plugins/frontend/backstage-plugin-iframe/README.md

```diff
@@ -0,0 +1,73 @@
+# Iframe plugin
+
+## Features
+
+- Displays a simple iframe from a url
+
+## Plugin Setup
+
+1. If you have standalone app (you didn't clone this repo), then in the [packages/app](https://github.com/backstage/backstage/blob/master/packages/app/) directory of your backstage instance, add the plugin as a package.json dependency:
+
+```bash
+yarn add @roadiehq/backstage-plugin-iframe
+```
+
+2. Import the plugin to the [entityPage.tsx](https://github.com/backstage/backstage/blob/master/packages/app/src/components/catalog/EntityPage.tsx) source file:
+
+```ts
+// packages/app/src/components/catalog/EntityPage.tsx
+import { 
+  iframePlugin,
+  EntityIFrameCard,
+  EntityIFramePage,
+} from '@roadiehq/backstage-plugin-iframe';
+...
+
+const iframeProps = {
+  frames: [
+    {
+      src: "https://example.com"
+    }
+  ]
+}
+const serviceEntityPage = (
+  <EntityLayoutWrapper>
+    ...
+    <EntityLayout.Route 
+      path="/mycustom-iframes"
+      title="Iframes">
+        <EntityIFramePage {...iframeProps} />
+    </EntityLayout.Route>
+    ...
+  </EntityLayoutWrapper>
+);
+
+const iframeProps = {
+  src: "https://example.com"
+}
+
+const overviewContent = (
+  <Grid container spacing={3}>
+    ...
+    <Grid item md={8}>
+      <EntityIFrameCard {...iframeProps}/>
+    </Grid>
+    ...
+  </Grid>
+);
+```
+
+## Allowlisting
+
+This particular plugin supports allowlisting. What this means is you can add a domain to the plugin's configuration that will be verified during the creation of the plugins components.
+
+The config is like so:
```

Comment:
I would suggest making it a bit clearer where this is configured so maybe just adding a 
// app-config.yaml here or in description above :)

Reply:
Thank you Irma! That makes sense